youtube_dl/extractor/sverigesradio.py

```diff
@@ -0,0 +1,130 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    int_or_none,
+)
+
+
+class SverigesRadioBaseIE(InfoExtractor):
+    _BASE_URL = 'https://sverigesradio.se/sida/playerajax'
+    _QUALITIES = ['high', 'medium', 'low']
+    _TYPES = [
+        {'type': 'audio', 'fatal': True},
+        {'type': 'secondary', 'fatal': False},
+    ]
+    _CODING_FORMATS = {
+        5: {'acodec': 'mp3', 'abr': 128},
+        11: {'acodec': 'aac', 'abr': 192},
+        12: {'acodec': 'aac', 'abr': 32},
+        13: {'acodec': 'aac', 'abr': 96},
+    }
+
+    def _extract_formats(self, query, audio_id, first_type, secondary_type):
+        audio_types = [{'type': first_type, 'fatal': True}]
+        if secondary_type:
+            audio_types += [{'type': secondary_type, 'fatal': False}]
+
+        audiourls = {}
+        for audio_type in audio_types:
+            for quality in self._QUALITIES:
+                audiourl = self._download_json(
+                    self._BASE_URL + '/getaudiourl', audio_id,
+                    fatal=audio_type['fatal'],
+                    query=dict(query, type=audio_type['type'], quality=quality, format='iis'))
+                if audiourl is None:
+                    continue
+
+                # for some reason url can be empty, skip if so
+                # also skip if url has already been seen (quality parameter is ignored?)
+                url = audiourl['audioUrl']
+                if url == "" or url in audiourls:
+                    continue
+
+                format = {'vcodec': 'none', 'url': url}
```

Comment:
do not use names of Python built-in functions(https://docs.python.org/3/library/functions.html#format).